From d67ffa1c823c447f4a12cfe567beeef473c3d5ff Mon Sep 17 00:00:00 2001
From: Colin Xu <colin.xu@intel.com>
Date: Tue, 30 Jun 2020 10:17:50 +0800
Subject: [PATCH 16/24] drm/i915/gvt: Move "guest page write error" to
 gvt_dbg_mm.

Not all "guest page write error" indicates gvt error. Some guest driver
may use previous tracked page memory for other purpose without notifying
gvt to disable the page tracker. There are other messages help
understand the status so suppress this to gvt_dbg_mm.

Signed-off-by: Colin Xu <colin.xu@intel.com>
(cherry picked from commit 6aa8dc9cb85cc438af2a7b15ebf44641a42f69fd)
Signed-off-by: Colin Xu <colin.xu@intel.com>
---
 drivers/gpu/drm/i915/gvt/page_track.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/gvt/page_track.c b/drivers/gpu/drm/i915/gvt/page_track.c
index 84856022528e..8c0f6708cecd 100644
--- a/drivers/gpu/drm/i915/gvt/page_track.c
+++ b/drivers/gpu/drm/i915/gvt/page_track.c
@@ -176,7 +176,7 @@ int intel_vgpu_page_track_handler(struct intel_vgpu *vgpu, u64 gpa,
 	} else {
 		ret = page_track->handler(page_track, gpa, data, bytes);
 		if (ret)
-			gvt_err("guest page write error, gpa %llx\n", gpa);
+			gvt_dbg_mm("guest page write error, gpa %llx\n", gpa);
 	}
 
 out:
-- 
2.17.1

